scanners/flutter/flutter.go

```diff
@@ -338,76 +294,70 @@ func getBuildablePlatform(hasAndroidProject, hasIosProject bool) string {
 
-		for _, platform := range platforms {
-			if platform != "none" {
-				if platform != "android" {
-					projectPathOption := models.NewOption(ios.ProjectPathInputTitle, ios.ProjectPathInputSummary, ios.ProjectPathInputEnvKey, models.TypeUserInput)
-					flutterPlatformOption.AddOption(platform, projectPathOption)
+	for _, platform := range platforms {
+		if platform != "android" {
+			projectPathOption := models.NewOption(ios.ProjectPathInputTitle, ios.ProjectPathInputSummary, ios.ProjectPathInputEnvKey, models.TypeUserInput)
+			flutterPlatformOption.AddOption(platform, projectPathOption)
 
-					schemeOption := models.NewOption(ios.SchemeInputTitle, ios.SchemeInputSummary, ios.SchemeInputEnvKey, models.TypeUserInput)
-					projectPathOption.AddOption("", schemeOption)
+			schemeOption := models.NewOption(ios.SchemeInputTitle, ios.SchemeInputSummary, ios.SchemeInputEnvKey, models.TypeUserInput)
+			projectPathOption.AddOption("", schemeOption)
 
-					distributionMethodOption := models.NewOption(ios.DistributionMethodInputTitle, ios.DistributionMethodInputSummary, ios.DistributionMethodEnvKey, models.TypeSelector)
-					schemeOption.AddOption("", distributionMethodOption)
+			distributionMethodOption := models.NewOption(ios.DistributionMethodInputTitle, ios.DistributionMethodInputSummary, ios.DistributionMethodEnvKey, models.TypeSelector)
+			schemeOption.AddOption("", distributionMethodOption)
 
-					for _, exportMethod := range ios.IosExportMethods {
-						configOption := models.NewConfigOption(cfg+"-app-"+platform, nil)
-						distributionMethodOption.AddConfig(exportMethod, configOption)
-					}
-				} else {
-					configOption := models.NewConfigOption(cfg+"-app-"+platform, nil)
-					flutterPlatformOption.AddConfig(platform, configOption)
-				}
-			} else {
-				configOption := models.NewConfigOption(cfg, nil)
-				flutterPlatformOption.AddConfig(platform, configOption)
+			for _, exportMethod := range ios.IosExportMethods {
+				configOption := models.NewConfigOption(cfg+"-app-"+platform, nil)
+				distributionMethodOption.AddConfig(exportMethod, configOption)
 			}
+		} else {
+			configOption := models.NewConfigOption(cfg+"-app-"+platform, nil)
+			flutterPlatformOption.AddConfig(platform, configOption)
 		}
 	}
 
 	return *flutterProjectLocationOption
 }
 
 // Configs ...
-func (scanner *Scanner) Configs(_ bool) (models.BitriseConfigMap, error) {
-	return scanner.DefaultConfigs()
+func (scanner *Scanner) Configs(isPrivateRepository bool) (models.BitriseConfigMap, error) {
+	return generateConfigMap(isPrivateRepository)
 }
 
 // DefaultConfigs ...
 func (scanner Scanner) DefaultConfigs() (models.BitriseConfigMap, error) {
+	return generateConfigMap(false)
+}
+
+func generateConfigMap(isPrivateRepository bool) (models.BitriseConfigMap, error) {
 	configs := models.BitriseConfigMap{}
 
 	for _, variant := range []struct {
 		configID string
 		test     bool
-		deploy   bool
```

Comment:
This is not important anymore, we'll generate a deploy workflow in all cases, even if the project has no Android nor iOS platforms.